src/ast-to-react.js

```diff
@@ -2,12 +2,32 @@
 
 const React = require('react')
 const xtend = require('xtend')
+const visit = require('unist-util-visit')
+
+let root = '';
 
 function astToReact(node, options, parent = {}, index = 0) {
   const renderer = options.renderers[node.type]
 
+  if(node.type === 'root') {
```

Comment:
Nitpick, but coding style wants a space between the if and the parentheses opening